.travis.yml

```diff
@@ -11,6 +11,54 @@ env:
     - CONDA_PY=3.7
   global:
     - PYSAM_LINKING_TEST=1
+    - TWINE_USERNAME=
+    - secure: ''
```

Comment:
Here the output of:
```
travis encrypt 'TWINE_PASSWORD=escaped_PyPI_password'
```
needs to be added. See https://docs.travis-ci.com/user/encryption-keys#note-on-escaping-certain-symbols for details.

Reply:
Thanks, I will try to merge this this week.